packages/AlgernonTests.package/ALGTestUI.class/instance/testBreadcrumbDoesHanldeMouseEvents.st

```diff
@@ -0,0 +1,5 @@
+testing
+testBreadcrumbDoesHanldeMouseEvents
+	"for ticket 11"
+	self assert: (self algerInstance breadcrumb handlesMouseDown: nil).
```

Comment:
`algerInstance` is now `algInstance`. (Therefore another test is failing)

Reply:
That test was apearently deleted and persisted in my branch. We now have the ...doesNoteHandle...